mindsdb_native/libs/helpers/general_helpers.py

```diff
@@ -203,6 +203,12 @@ def evaluate_regression_accuracy(
         backend,
         **kwargs
     ):
+    if kwargs.get('ts_window', None) is not None and backend.predictor and \
+            kwargs['ts_window'] <= 2*len(predictions[column]):
+        # with enough rows, truncate 1st 'window' outputs to account for incomplete historical context
+        predictions[column] = predictions[column][kwargs['ts_window']:]
+        true_values = true_values[kwargs['ts_window']:]
```

Comment:
This shouldn't trigger for none-timesereis problems, as it stands it will. Also, why check `backend.predictor` ?

Reply:
Ok, `None` being the default avoids any non-TS task entering now. `backend.predictor` was a leftover, nice catch :+1: